source/renderer/app/components/hardware-wallet/layouts/HardwareWalletWithNavigation.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import React, { Component } from 'react';
+import type { Node } from 'react';
+import { observer } from 'mobx-react';
+import HardwareWalletNavigation from '../navigation/HardwareWalletNavigation';
+import styles from './HardwareWalletWithNavigation.scss';
+import ConnectHardwareWallet from '../settings/ConnectHardwareWallet';
+
+type Props = {
+  children?: Node,
+  activeItem: string,
+  hasNotification?: boolean,
+  walletNotConnected: boolean,
+  isActiveScreen: Function,
+  onOpenExternalLink: Function,
+  onWalletNavItemClick: Function,
+  isLedger: boolean,
+  isTrezor: boolean,
+  isDeviceConnected: boolean,
+  fetchingDevice: boolean,
+  exportingExtendedPublicKey: boolean,
+  isExportingPublicKeyAborted: boolean,
+};
+
+@observer
+export default class HardwareWalletWithNavigation extends Component<Props> {
+  render() {
+    const {
+      children,
+      activeItem,
+      hasNotification,
+      walletNotConnected,
+      isActiveScreen,
+      onWalletNavItemClick,
+      onOpenExternalLink,
+      isLedger,
+      isTrezor,
+      isDeviceConnected,
+      fetchingDevice,
+      exportingExtendedPublicKey,
+      isExportingPublicKeyAborted,
+    } = this.props;
+
+    return (
+      <div className={styles.component}>
+        {walletNotConnected ? (
+          <ConnectHardwareWallet
+            onOpenExternalLink={onOpenExternalLink}
+            isLedger={isLedger}
+            isTrezor={isTrezor}
```

Comment:
@DeeJayElly  If we are in the connecting process how we can now if HW is Ledger or Trezor?. We know that once device is connected.
cc @nikolaglumac